### ISSUE TITLE
How can I hide the original input element.

### ISSUE THREAD

**Issue by ilbeyataberkcan** [yairEO/tagify#961]
Hi everyone. I'm a newbie and want to use the tagify react for a project. However, I'm having a problem with it. 

There are two issues I have and  pretty sure both of them are stupid. 

-1: Tagify doesn't initialize with my modal. But it does when I change engage with other elements. I believe I can fix this with useEffect hook.

-2 I can't hide the original Input component.
![image](https://user-images.githubusercontent.com/32396853/146380115-c8da3356-08d8-4d09-b113-ea7d573056e3.png)

I've tried to use different css variables but it does effect the tagify <tag> element too.

This is how I initialize the tagify:
`    let tagifyInputComponent = (<input className="form-control" id="companies"/>);

    // @ts-ignore
    const tagify = new Tagify(document.querySelector("#companies"), {
        autoComplete: {
            enabled: false
        },
        // templates: {
        //     // Here we define how the tags will look like.
        // },
        dropdown: {
            enabled: 0, // suggest tags after a single character input
        },
        placeholder: "Enter assigned Companies",
        callbacks: {
            "change": (e: TagData) => console.log(e.detail)
        }

    })`

I also having an warning in the console:
![image](https://user-images.githubusercontent.com/32396853/146380593-419e6e12-3924-4d5d-9e96-166485f8f49f.png)

I'm using Microsoft Edge for development but I also tried the Google Chrome. Even though I don't think it's about the browser.

In short:
Can you please show me the correct way to do this? This was one of those moments where you start to believe in yourself but a simple thing destroys your confidence 😅

**Comment by yairEO** [comment]
did you import Tagify CSS globally on your project?

The original input is hidden by default, you don't need to do it yourself

**Comment by ilbeyataberkcan** [comment]
I'm using a template from Keenthemes I believe they handled it all. But for some reason it doesn't work

**Comment by ilbeyataberkcan** [comment]
I've changed the implementation to React Component. But the original input field still there:


![image](https://user-images.githubusercontent.com/32396853/146570650-cc372910-c9f5-4034-8a1c-36a1457047b9.png)

![image](https://user-images.githubusercontent.com/32396853/146570703-aa6acca8-6230-4e59-bada-b77bf12a2299.png)

**Comment by yairEO** [comment]
what about my first question regarding importing the CSS file?

**Comment by ilbeyataberkcan** [comment]
Template provider Keenthemes did import the css files globally. But when I import it manually it throws an error "undefined function math.div'

**Comment by yairEO** [comment]
you probably use the very old and very very deprecated `node-sass` node package.

Replace it with [sass](https://www.npmjs.com/package/sass)

You don't have to import the SCSS file. you can import the compiled CSS file instead. I still don't understand if this is a question about this template or regardless..

**Comment by ilbeyataberkcan** [comment]
Thanks I'll try and let you know. I've tried to update it before but I revert it because some other problems occurred.

And no it's not about template. The template docs shows the basic implementation of tagify but it doesn't work for me. Since this isn't the first difficulty that I encountered with this template: I'm trying to fix the issues by going to basics. I'm stating the template because there is many new things for me. As I said: I'm new to react and still learning the advanced stuff.

Thank you so much for your help. I at least have lead to where I should be looking.

**Comment by yairEO** [comment]
this template doesn't have a github page where you can write the issue there? or contact the author of the template?
